pages/ecosystem/index.vue

```diff
@@ -168,6 +170,16 @@ import { GeneralLink } from '~/constants/appLinks'
       }
     },
 
+    getTierNames (tiers: any) {
+      const tierNames = tiers.map((tier: any) => tier.name)
+      return tierNames
+    },
+
+    getTierDescription (tier: string) {
+      const tip = (this as any).tiers.find((tip: any) => tip.name === tier)
+      return tip.description
```

Comment:
Sorry, I've forgotten this one. Please, add an option just in case there are no description: `return tip.description || ""` , for example